applications/fluid_structure_interaction_precice/cylinder_with_flag/input.json

```diff
@@ -0,0 +1,28 @@
+{
+    "General": {
+        "Precision": "double",
+        "Dim": "2",
+        "IsTest": "false"
+    },
+    "SpatialResolution": {
+        "DegreeFluid": "6",
+        "DegreeStructure": "2",
+        "RefineFluid": "0",
+        "RefineStructure": "0"
+    },
+    "FSI" : {
+        "Method": "IQN-ILS",
+        "AbsTol": "1.e-8",
+        "RelTol": "1.e-3", 
+        "OmegaInit": "0.3",
+        "ReusedTimeSteps": "8",
+        "PartitionedIterMax": "100"
+    },
+    "Application" : {
+    },
+    "Output": {
+        "OutputDirectory": "output/cylinder_with_flag/",
+        "OutputName": "test",
+        "WriteOutput": "false"
+  }
+}
```

Comment:
Why do you use the "normal" ExaDG input file here, while you have two files "fluid-exadg" and "solid-exadg" for the new "perpendicular_flap" example?

Are the FSI-parameters used here? You said in the PR description that strong-coupling is not used, so that the FSI parameters should not be relevant. If so, we should remove them, and introduce them again once they are required.